app/components/swarmVisualizer/swarmVisualizer.html

```diff
@@ -55,6 +55,42 @@
   </div>
 </div>
 
+<div class="row">
+  <div class="col-sm-12">
+    <rd-widget>
+      <rd-widget-header icon="fa-info-circle" title="About Cluster Visualizer">
+      </rd-widget-header>
+      <rd-widget-body>
+        <form class="form-horizontal">
+          <div class="form-group">
+            <div class="col-sm-12">
+              <span class="small text-muted">
+                This view displays real-time information about the Swarm.
+              </span>
+            </div>
+          </div>
+          <div class="form-group">
```

Comment:
Rather than seeing a new panel in this view, I'd prefer to have the refresh rate related fields available under a new section called "Refresh" under the "Filters" section inside the Cluster information panel.

Remove the "This view displays real-time information about the Swarm." message as well.